client/store/helpers/actions.js

```diff
@@ -20,6 +20,8 @@ export default function ($apiUrl) {
     return api.fetch(opts).then(res => commit('reset', res));
   };
 
+  const add = ({ commit }, model) => commit('add', model);
```

Comment:
Added this so custom containers can add elements to the store without them being saved.

Reply:
@abasic Don't we have add mutation already. What's the drive for action?

Reply:
@underscope The drive was to avoid calling the mutation directly but go through the full flow of calling an action that calls the mutation. I know it is not necessary to get the desired result but wanted to keep it inline with all the other event handlers that are there.